builtin/logical/pki/path_fetch.go

```diff
@@ -96,6 +110,37 @@ func (b *backend) pathFetchCertList(req *logical.Request, data *framework.FieldD
 	return logical.ListResponse(entries), nil
 }
 
+func (b *backend) pathFetchReadCAChain(req *logical.Request, data *framework.FieldData) (*logical.Response, error) {
+	caInfo, err := fetchCAInfo(req)
+	switch err.(type) {
+	case errutil.UserError:
+		return nil,
+			errutil.UserError{Err: fmt.Sprintf("Could not fetch the CA certificate: %s", err)}
+	case errutil.InternalError:
+		return nil,
+			errutil.InternalError{Err: fmt.Sprintf("Error fetching CA certificate: %s", err)}
+	}
+
+	caChain := caInfo.GetCAChain()
```

Comment:
Oh, you're right -- a GET will return a 404 on no data returned, unlike a POST or DELETE.

Keeping the way it is sounds fine but if there's no data we should return 204, not 200.